src/humancellatlas/data/metadata/helpers/schema_validation.py

```diff
@@ -0,0 +1,48 @@
+from functools import (
+    lru_cache,
+)
+import json
+import logging
+
+from jsonschema import (
+    FormatChecker,
+    ValidationError,
+    validate,
+)
+import requests
+
+from humancellatlas.data.metadata.api import (
+    JSON,
+)
+from humancellatlas.data.metadata.helpers.exception import (
+    RequirementError,
+)
+
+logger = logging.getLogger(__name__)
+
+
+class SchemaValidator:
+    _instance = None  # singleton object
+
+    def __new__(cls):
+        if cls._instance is None:
+            cls._instance = super(SchemaValidator, cls).__new__(cls)
+        return cls._instance
```

Comment:
Sorry, I meant quasi-singleton like we use them in Azul (controllers, services). My mistake.

```suggestion
```